packages/gatsby-source-graphcms/src/util.js

```diff
@@ -0,0 +1,29 @@
+import R from 'ramda'
+
+
+// If type ends in a non-vowel, we need to append es. Else s.
+export const formatTypeName = t => `all${t}${/s$/.test(t) ? `es` : `s`}`
```

Comment:
perhaps should use this instead? https://www.npmjs.com/package/pluralize

Or is this what GraphCMS says is their process?